tests/mock/dist.rs

```diff
@@ -356,6 +364,13 @@ impl MockDistServer {
                     );
                     toml_target.insert(String::from("xz_hash"), toml::Value::String(xz_hash));
                 }
+                if let Some(zst_hash) = hash.zst {
```

Comment:
Can't the dist abstraction write this out ?

Reply:
I'll admit I lazily extended what existed already.  We can certainly log to refactor this part of the tests.